src/java.desktop/share/classes/sun/awt/FontConfiguration.java

```diff
@@ -1244,15 +1244,23 @@ public String getFileNameFromPlatformName(String platformName) {
         return filenamesMap.get(platformName);
     }
 
+    private static final String fontconfigErrorMessage = "Fontconfig head is null, check your fonts or fonts configuration";
```

Comment:
```suggestion
    private static final String fontconfigErrorMessage =
            "Fontconfig head is null, check your fonts or fonts configuration";
```
This way it fits into 80-column limit; otherwise the line is too long: 124-columns.